cmd/crictl/sandbox.go

```diff
@@ -322,7 +374,7 @@ var listPodCommand = &cli.Command{
 		if err != nil {
 			return err
 		}
-		if err = ListPodSandboxes(runtimeClient, opts); err != nil {
+		if err = OutputPodSandboxes(runtimeClient, opts); err != nil {
```

Comment:
in case of various filters, potential improvement over time may be to show filter-specific additional column. For since it may be time passed since that value in the filter, for example. This is just a nit.